jsettlers.main.android/src/main/res/values/strings.xml

```diff
@@ -134,6 +134,11 @@
     <string name="game_menu_unpause">Unpause</string>
     <string name="game_menu_paused">Paused</string>
     <string name="game_menu_saved">Game saved</string>
+    <string name="game_menu_skip_minute">Skip 1 minute</string>
```

Comment:
The text for this was previously stored in the string `skip_1_minute`. Please remove that string, which is now unused and use it for the localization with the other languages.